src/cript/nodes/uuid_base.py

```diff
@@ -52,7 +52,7 @@ def url(self):
         from cript.api.api import _get_global_cached_api
 
         api = _get_global_cached_api()
-        return f"{api.host}/{self.uuid}"
+        return f"{api.host}/{api.api_prefix}/{api.api_version}/{self.uuid}"
```

Comment:
ahh ok this compensates for it being removed in the other part of the codebase

Reply:
yes!